Classes/Repository/RepositoryBranches/RepositoryBranchesViewController.swift

```diff
@@ -69,22 +69,24 @@ RepositoryBranchSectionControllerDelegate {
     }
 
     override func fetch(page: String?) {
-        client.fetchRepositoryBranches(owner: owner,
-                                 repo: repo
-            ) {  [weak self] result in
+        client.fetchRepositoryBranches(
+            owner: owner,
+            repo: repo,
+            nextPage: page as String?
+        ) {  [weak self] result in
             guard let strongSelf = self else { return }
             switch result {
-            case .success(let branches):
+            case .success(let payload):
                 self?.branches = RepositoryBranchesViewController.arrangeBranches(
```

Comment:
Any reason not to use `strongSelf` here and in other places where `self?` is used?

Reply:
Didn’t notice the discrepancy in that code block. No reason at all, I can change and use `strongSelf`.

Reply:
Yeah that'd be nice. Then I will merge after :)